search-app/src/cmr/search/api/services.clj

```diff
@@ -8,6 +8,8 @@
 (def service-api-routes
   (context "/services" []
 
+    ;; Search for services route is defined in routes.clj
```

Comment:
Why?

Reply:
search by concept types is handled generically (sort of) in routes.clj for all concept types.